projboard/models.py

```diff
@@ -19,11 +19,68 @@ def get_subject_by_name(name):
         :return: Subject object
         """
         try:
-            subject = User.objects.get(name=name)
+            subject = Subject.objects.get(name=name)
         except ObjectDoesNotExist:
             return None
         return subject
 
+    def __str__(self):
+        return f"{self.name}"
+
+    @staticmethod
+    def get_list_subjects_names():
+        """
+        Method get list subject names
+        :return: the list of subject
+        """
+        subjects_list = list(Subject.objects.all().values_list('name', flat=True))
+        return subjects_list
+
+    @staticmethod
+    def create_new_subject(name):
```

Comment:
This method is not only creating a new subject, it also returns an existing subject (if exist).  
Consider renaming the method to `get_subject`

Reply:
I changed to "create_and_get_subject"